hardware-and-software-requirements.md

```diff
@@ -18,7 +18,9 @@ TiDB 作为一款开源分布式 NewSQL 数据库，可以很好地部署和运
     |  操作系统   |   支持的 CPU 架构   |
     |   :---   |   :---   |
     | <ul><li>Red Hat Enterprise Linux 8.4 及以上的 8.x 版本</li><li>CentOS 8.4 及以上的 8.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
+    | Red Hat Enterprise Linux 8 Stream |  <ul><li>x86_64</li><li>ARM 64</li></ul>  |
     | <ul><li>Red Hat Enterprise Linux 7.3 及以上的 7.x 版本</li><li>CentOS 7.3 及以上的 7.x 版本</li></ul>  |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
+    |  AWS Linux         |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
```

Comment:
@easonn7 PTAL

Reply:
```suggestion
    |  Amazon Linux 2         |  <ul><li>x86_64</li><li>ARM 64</li></ul>   |
```

- "AWS Linux" doesn't exist. "Amazon Linux" does.
- For "Amazon Linux" we should only support version 2 as there are huge differences between Amazon Linux 2 and Amazon Linux 2022. While Amazon Linux 2 is compatible with RHEL7/CentOS7, Amazon Linux 2022 is built with Fedora Linux as a base, but with many big enhancements on top of that.

References:

- https://aws.amazon.com/linux/amazon-linux-2022/
- https://aws.amazon.com/amazon-linux-2/
- https://docs.aws.amazon.com/linux/al2022/ug/compare-al2-to-AL2022.html